### PR TITLE
Remove commenting settings from the banner page model

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

**Comment by saeedmb8** [comment]
@justinvdm tiny review Please :)

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/static/css/_admin_css.scss [hunk 1]
```diff
@@ -1,5 +1,8 @@
 .versions{
   color: black;
+  a {
+    color: red;
+  }
```
Comment (justinvdm) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/static/css/_admin_css.scss]: Could you explain why this is necessary? If it is, would it be better to go with the [reds wagtail uses](https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailadmin/static_src/wagtailadmin/scss/_variables.scss#L32-L36) for consistency?

File: molo/core/models.py [hunk 2]
```diff
@@ -147,14 +147,7 @@ class BannerPage(TranslatablePageMixin, Page):
...
     FieldPanel('title', classname='full title'),
     ImageChooserPanel('banner'),
-    PageChooserPanel('banner_link_page'),
-    MultiFieldPanel(
-        [
-            FieldPanel('commenting_state'),
-            FieldPanel('commenting_open_time'),
-            FieldPanel('commenting_close_time'),
-        ],
-        heading="Commenting Settings",)
```
Comment (justinvdm) [inline molo/core/models.py]: Could you explain a bit why we are removing this?

**Comment by justinvdm** [comment]
Two minor comments, otherwise :+1:.

File: molo/core/models.py (hunk 2 above)
Comment (saeedmb8) [inline molo/core/models.py]: @justinvdm Commenting Settings has been added by mistake to banner pages, a banner page doesn't have commenting option, basically a user can't comment on a banner, so doesn't need commenting settings

Comment (justinvdm) [inline molo/core/models.py]: Ah, thanks :)

**Comment by justinvdm** [comment]
:+1: once travis is happy.

**Pull request merged**